01-front-end-basics/html-forms/25/solutions.md

```diff
@@ -0,0 +1,104 @@
+**Forms & Queries** -- Solutions
+
+## Query Parameters
+
+#### convertToObject(query_parameter_string)
+``` javascript
+// var string = "a=apple&b=banana&c=cola&d=duck&e=egads&f=fancy";
+function convertToObject(qp) {
+    var output = {};
+    if (!qp) { return output; }
+    qp.split("&").forEach(function(sub_str){
```

Comment:
Are we ready for anonymous functions?  I intro'd them today and saw it starting to make sense...but....